src/components/Kv/KvMultiCodeBlock.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+	<div>
+		<div class="tab-container">
+			<div
+				v-for="{title, index} in code"
+				:key="index"
+				class="tab"
+				:class="`${isTabSelected(index) ? 'active' : ''}`"
+				@click="setSelected(index)"
+			>
+				{{ title }}
+			</div>
+		</div>
+		<kv-code-block :code="selectedCode" :nowrap="nowrap" />
+	</div>
+</template>
+
+<script>
+import KvCodeBlock from '@/components/Kv/KvCodeBlock';
+
+export default {
+	components: {
+		KvCodeBlock,
+	},
+	data() {
+		return {
+			code: [
```

Comment:
Can we move this out into the parent that "uses" the component?

Reply:
Done :+1: